src/core/agent/services/connectionService.ts

```diff
@@ -171,6 +171,16 @@ class ConnectionService extends AgentService {
       connectionMetadata.pending = false;
       connectionMetadata.status = ConnectionStatus.CONFIRMED;
     }
+    this.props.eventEmitter.emit<ConnectionStateChangedEvent>({
```

Comment:
This was in the else statement before. Can you explain why it was removed, or why it was re-added like this?

Reply:
sorry, maybe this is some mistake from my side, if I do not add this, the connection will not appear until resolveOobi is done. So I think it necessary to display a connection pending before the Oobi is resolved